README.md

```diff
@@ -2,42 +2,35 @@
   <img src="logo.png">
 </p>
 
+> 📣🆕 [https://api.sorare.com/federation/graphql](https://api.sorare.com/federation/graphql) is the new Sorare API endpoint. It's a federated superset of existing API endpoints so all existing queries remain valid, just replace the URL in your application. Visit the [playground](https://api.sorare.com/federation/graphql/playground).
+
 # Sorare API
 
 At Sorare, we are committed to providing an open platform for developers to build upon.
 
 While our Cards are stored on the Ethereum blockchain (or within a [Starkware rollup](https://starkware.co/starkex/)) we support an API that provides more detailed information.
 
-The Sorare APIs are provided by [GraphQL](https://graphql.org/). Sorare provides distinct APIs for **Sorare: Football**, **Sorare: MLB** and **Sorare: NBA**. We will move towards a [federated GraphQL API](https://www.apollographql.com/docs/federation/) in the near future.
-
-The **Sorare: Football** API is hosted on [https://api.sorare.com/graphql](https://api.sorare.com/graphql). The documentation can be found under the Docs section of the [Football GraphQL playground](https://api.sorare.com/graphql/playground).
-
-The **Sorare: MLB** and **Sorare: NBA** APIs are hosted on [https://api.sorare.com/sports/graphql](https://api.sorare.com/sports/graphql). The documentation can be found under the Docs section of the [Sports GraphQL playground](https://api.sorare.com/sports/graphql/playground).
+The Sorare API are provided by [GraphQL](https://graphql.org/). Sorare provides a federated API for all sports. The API is hosted on [https://api.sorare.com/federation/graphql](https://api.sorare.com/federation/graphql). The documentation can be found under the Docs section of the [GraphQL playground](https://api.sorare.com/federation/graphql/playground).
 
-You can easily download the GraphQL schema using `[@apollo/rover](https://www.apollographql.com/docs/rover/)`:
+You can easily download the GraphQL schema using [@apollo/rover](https://www.apollographql.com/docs/rover/):
 
 ```bash
-$ npx -p @apollo/rover rover graph introspect https://api.sorare.com/graphql > schema-football.graphql
-$ npx -p @apollo/rover rover graph introspect https://api.sorare.com/sports/graphql > schema-us-sports.graphql
+$ npx -p @apollo/rover rover graph introspect https://api.sorare.com/graphql > schema.graphql
 ```
 
-## Federating APIs
-
-The original [https://api.sorare.com/graphql](https://api.sorare.com/graphql) API is providing all the sport-agnostic GraphQL types, fields and subscriptions which allows you to manipulate Football, MLB and NBA NFTs.
+## Federated API
 
 ### Football
 
-You have access to the Football-specific `Card`, `Player`, `So5Fixture`, `So5Leaderboard`, ... resources through the [https://api.sorare.com/graphql](https://api.sorare.com/graphql) API.
+The Football-specific resources are either not prefixed or prefixed with `So5`: `Card`, `Player`, `So5Fixture`, `So5Leaderboard`, etc.
```

Comment:
Football resources have moved to the football subroot.

Reply:
I added a sentence